### PR TITLE
6.7 indriect_geometry.rst release notes

### PR DESCRIPTION
**Pull request by jhaigh0**
**Description of work**

Edit the 6.7 indirect geometry release notes making changes for grammar, spelling and relevance.

Note that empty sections are not removed until just before release to accommodate any last minute changes.

**To test:**

 - Build `docs-html`
 - Open the html file `/build/docs/release/v6.7.0/indirect_geometry.html`
 - Check that all release notes make sense to you
 - Check that links go to the correct place
 - Be picky about stylistic consistencies

---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Review by MohamedAlmaki — CHANGES_REQUESTED** [review]
Just a few comments on small things

File: docs/source/release/v6.7.0/indirect_geometry.rst [hunk 1]
```diff
@@ -7,30 +7,30 @@ Indirect Geometry Changes
...
-- Fixed a hard crash caused by attempted loading of empty list of enabled SD detectors of IN16B data in :ref:`LoadILLIndirect <algm-LoadILLIndirect>`.
-- Fixed a bug that meant when the workspace attribute of a function was changed (e.g. resolution or tabulated function) the function was not updated. This would lead to a crash as Mantid believed that the option was invalid.
-- Fixed a hard crash caused by highlighting more cells than rows in the data input table and then pressing Remove.
+- A bug has been fixed that caused the :ref:`Elwin <elwin>` tab to not run specific spectra when using the workspace input method.
+- Fixed a hard crash caused by attempting to load an empty list of enabled SD detectors of IN16B data in :ref:`LoadILLIndirect <algm-LoadILLIndirect>`.
+- Fixed a bug that meant when the workspace attribute of a function was changed, (e.g. resolution or tabulated function) the function was not updated. This would lead to a crash as Mantid believed that the option was invalid.
+- Fixed a hard crash caused by highlighting more cells than rows in the data input table and then pressing ``Remove``.
```
[the remark this answers is not in the document]
Comment (jhaigh0) [reply docs/source/release/v6.7.0/indirect_geometry.rst:20]: Do you mean the note on line 17? I think it is okay as includes what's relevant to the user. 'Specific spectra' is a little vague, though.

[the remark this answers is not in the document]
Comment (MohamedAlmaki) [reply docs/source/release/v6.7.0/indirect_geometry.rst:20]: I mean in line 20

[the remark this answers is not in the document]
Comment (jhaigh0) [reply docs/source/release/v6.7.0/indirect_geometry.rst:20]: Ah, that's the ConvFit tab in the indirect interface. I can update that note to be clearer.

[the remark this answers is not in the document]
Comment (MohamedAlmaki) [reply docs/source/release/v6.7.0/indirect_geometry.rst]: ok, thanks

**Review by MohamedAlmaki — APPROVED** [review]
Looks good, thanks for the changes

**Pull request merged**